src/main/java/uk/gov/companieshouse/api/testdata/service/impl/RandomServiceImpl.java

```diff
@@ -46,4 +46,11 @@ public String getEncodedIdWithSalt(int idLength, int saltLength) {
     public String getEtag() {
         return GenerateEtagUtil.generateEtag();
     }
+
+    @Override
+    public String addSaltAndEncode(String baseString, int saltLength) {
```

Comment:
Maybe we should call this method from `getEncodedIdWithSalt` to remove code duplication